systemtests/ldap_management_test.go

```diff
@@ -8,13 +8,13 @@ import (
 )
 
 var (
-	ldapConfig = `{"server":"localhost", 
-              "port":4563, 
```

Comment:
We're not testing connection here. Its just testing LDAP endpoints. So, it doesn't matter.